cra/sample/pr-trigger.yaml

```diff
@@ -0,0 +1,108 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: TriggerTemplate
+metadata:
+  name: pr-template
+spec:
+  params:
+    - name: repository
+      description: The git repo
+    - name: branch
+      description: branch
+    - name: target-branch
+      description: target branch
+    - name: target-commit-id
+      description: target commit id      
+    - name: repository-integration
+      description: the repo integration name
+    - name: api-key
+      description: the ibmcloud api key
+    - name: commit-id
+      description: commit id
+    - name: commit-timestamp
+      description: commit timestamp
+    - name: pipeline-debug
+      description: toggles debug mode for the pipeline
+    - name: pr-url
+      description: PR url
+    - name: pr-number
+      description: PR number  
+    - name: pr-name
+      description: PR name           
+    - name: artifactory-dockerconfigjson
+      description: dockerconfigjson for pullsecrets
+    - name: scm-type
+      description: source code type used (github, github-ent, gitlab)
+    - name: project-id
+      description: project id
+      default: ""             
+
+  resourcetemplates:
+    - apiVersion: v1
+      kind: PersistentVolumeClaim
+      metadata:
+        name: pipelinerun-$(uid)-pvc
+      spec:
+        resources:
+          requests:
+            storage: 5Gi
+        volumeMode: Filesystem
+        accessModes:
+          - ReadWriteOnce
+    - apiVersion: v1
+      kind: Secret
+      data:
+        .dockerconfigjson: $(params.artifactory-dockerconfigjson)
+      metadata:
+        name: artifactory-pull-secret
+      type: kubernetes.io/dockerconfigjson
+    - apiVersion: v1
+      kind: ServiceAccount
+      imagePullSecrets:
+        - name: artifactory-pull-secret
+      metadata:
+        name: sa-pullsecret
+    - apiVersion: v1
+      kind: Secret
+      metadata:
```

Comment:
is a specific/deciated secret required in the sample ?
why do not use the secure-properties secret that is created for us by CD Tekton Support - cf https://cloud.ibm.com/docs/ContinuousDelivery?topic=ContinuousDelivery-tekton_environment#tekton_envprop